src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  --hover-active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.header__img_logo {
+  margin: 10px 0;
+  cursor: pointer;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  list-style: none;
+  display: flex;
+}
+
+.nav__item:nth-of-type(n + 2) {
+  margin-left: 20px;
+}
+
+.nav__item_link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__item_link:hover,
+.is-active {
+  color: var(--hover-active-color);
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: relative;
+  bottom: 4px;
+  height: 4px;
+  border-radius: 2px;
+  background-color: var(--hover-active-color);
+}
```

Comment:
What do you think about such structure?

```
.nav__item {
  height: 64px;
  position: relative;
}

.nav__item:first-of-type {
  margin-left: 20px;
}

.nav__item_link {
  height: 100%;
  display: flex;
  align-items: center;
  text-decoration: none;
  color: black;
}

.nav__item_link:hover {
  color: var(--hover-active-color);
}

.is-active {
  color: var(--hover-active-color);
}

.nav__item_link.is-active::after {
  content: "";
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  height: 4px;
  border-radius: 2px;
  background-color: var(--hover-active-color);
}
```

Reply:
`.nav__item {
  height: 64px;
  position: relative;
}`
height should be 60 px

`.nav__item:first-of-type {
  margin-left: 20px;
}`
I think you mean here :not(first-of-type)

Yea I think position absolute is better. I don't have to take into account this ::after height. Thanks for the insights!
I added external reset.css file as you suggested with [Meyer reset](https://meyerweb.com/eric/tools/css/reset/) but ES lint is giving me hard time with that 😂

Reply:
- Yup now I see the design in Figma says 60px
- Also I ment something like
```
.nav__item {
  height: 64px;
  position: relative;
  margin-left: 20px;
}

.nav__item:first-of-type /* or :first-child */ {
  margin-left: 0;
}
```
My bad with the advice, reviewing css is always challenging as you are not actually seeing the results 😅
- I would import the reset pretty similarly
- also for the future if you have more than 2 css files you may create one that only imports the others then you only insert that one to the html -> however that's my personal preference